modules/checklist/assets/js/components/editor-v-2.js

```diff
@@ -0,0 +1,45 @@
+import { Badge } from '@elementor/ui';
+import { __ } from '@wordpress/i18n';
+import * as React from 'react';
+import { App } from './app';
+import RocketIcon from '@elementor/icons/RocketIcon';
+
+const IconWithBadge = ( { invisible } ) => {
+	return (
+		<Badge color="primary" variant="dot" invisible={ invisible }>
+			<RocketIcon />
+		</Badge>
+	);
+};
+
+IconWithBadge.propTypes = {
+	invisible: PropTypes.bool,
+};
+
+export const editorV2 = () => {
+	const { utilitiesMenu } = window.elementorV2.editorAppBar;
+
+	utilitiesMenu.registerLink( {
+		id: 'app-bar-menu-item-checklist',
+		priority: 5,
+		useProps: () => {
+			return {
+				title: __( 'Checklist', 'elementor' ),
+				icon: () => <RocketIcon />,
+				onClick: () => {
+					elementor.editorEvents.dispatchEvent(
+						elementor.editorEvents.config.names.topBar.checklist,
```

Comment:
@mykytamurzin Just making sure these events exist?

Reply:
@louiswol94 should be added here `modules/editor-events/assets/js/editor/events-config.js`

Reply:
Do we even want this to be tracked?